### PR TITLE
Add use instructions to README

### PR DESCRIPTION
**Pull request by rivertam**
Sorry about the formatting -- Prettier did it. I'll turn it off if it
gets annoying in actual docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+- [Chat on gitter](https://gitter.im/actix/actix)
+
+## Using this library
+
+This repository is an [`mdBook`](https://github.com/rust-lang-nursery/mdBook)
+project. To use it for this project:
+
+- Install `mdBook` if you haven't already: `cargo install mdbook`
+- In the `actix` directory: `mdbook watch -o`
+  - This automatically opens your browser and watches the md files for changing
```
Comment (DoumanAsh) [inline README.md:20]: I don't think you need nested list `This automatically` just move it outer sentence?

Comment (rivertam) [reply README.md:20]: I don't know how much it matters but I like to think the outer list is a list of instructions and the inner lists are explanations. I don't really need a list really I guess.

Comment (rivertam) [reply README.md:20]: Regardless I should also clarify you'll still need to refresh the page

Comment (DoumanAsh) [reply README.md:20]: Ok, let me know once you'll clarify it and I'll merge

Comment (rivertam) [reply README.md:20]: Clarified. I kept it in the inner list. Let me know if you still want me to change that.

**Review by DoumanAsh — APPROVED** [review]

**Pull request merged**